patches/extra/ungoogled-chromium/allow-thinlto-optimizations.patch

```diff
@@ -0,0 +1,17 @@
+From: Marek Behún <kabel@kernel.org>
+
+Use -Wl,--lto-O2 optimizations instead of just -Wl,--lto-O0 even if not
+official build.
+
+diff -Naurp a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
+--- a/build/config/compiler/BUILD.gn
++++ b/build/config/compiler/BUILD.gn
+@@ -129,7 +129,7 @@ declare_args() {
+   # the space overhead is too great. We should use some mixture of profiles and
+   # optimization settings to better tune the size increase.
+   thin_lto_enable_optimizations =
+-      (is_chromeos_ash || is_android || is_win || is_linux) && is_official_build
```

Comment:
This left out Debug builds with optimizations too. Chromium optimization has threee levels instead fo two, which are debug, non-debug non-official and official.